solana/system_program.py

```diff
@@ -304,14 +335,50 @@ def sys_program_id() -> PublicKey:
     return PublicKey("11111111111111111111111111111111")
 
 
-def create_account(param: CreateAccountParams) -> Transaction:
+def create_account(params: CreateAccountParams) -> Transaction:
     """Generate a Transaction that creates a new account."""
-    raise NotImplementedError("create_account not implemented")
```

Comment:
Including an example on how to call this function would be nice,

i.e.
https://github.com/michaelhly/solana-py/blob/d636be38aba6d7543b494335a18e904d43024c5a/solana/system_program.py#L387

Reply:
added